domains/cemalgnlts.json

```diff
@@ -4,6 +4,8 @@
     "discord": "cemalgnlts"
   },
   "record": {
-    "CNAME": "cemalgnlts.surge.sh"
+    "CNAME": "cemalgnlts.surge.sh",
```

Comment:
The test failed because you combined CNAME record with other records. See https://docs.is-a.dev/domain-structure/#record-required
> CNAME record: This must be a hostname (something.tld). It cannot be used in conjunction with any other record types unless your domain is proxied